hack/remote-extension-up.sh

```diff
@@ -38,7 +38,7 @@ else
 fi
 
 echo "Deploying registry-cache admission in garden cluster"
-SKAFFOLD_DEFAULT_REPO=garden.local.gardener.cloud:5001 SKAFFOLD_PUSH=true skaffold run -m admission -p remote-extensions
+SKAFFOLD_CHECK_CLUSTER_NODE_PLATFORMS=true SKAFFOLD_DEFAULT_REPO=registry.local.gardener.cloud:5000 SKAFFOLD_PUSH=true skaffold run -m admission -p remote-extensions
```

Comment:
See my previous comment. Can this be the default and not specified explicitly here?